crates/pgsrv/src/ssl.rs

```diff
@@ -148,54 +148,21 @@ mod tests {
 
     const TEST_CERT: &str = r#"
 -----BEGIN CERTIFICATE-----
-MIIDDTCCAfWgAwIBAgIUBhtwC7ezA9RQTLP8EsHdcGB8X+cwDQYJKoZIhvcNAQEL
-BQAwFjEUMBIGA1UEAwwLZ2xhcmVkYi5jb20wHhcNMjMwNTI1MDA0MDExWhcNMjQw
-NTI0MDA0MDExWjAWMRQwEgYDVQQDDAtnbGFyZWRiLmNvbTCCASIwDQYJKoZIhvcN
-AQEBBQADggEPADCCAQoCggEBALnqZnssy27imdBiD9IhVYengY0g5LEjR0RKDCbI
-iyxicWZNHrVurjXlm0XCEi1TX4BJR0l6UIN0Hbw9HnFC69zZw7dxRCe+1XIvj49B
-ZxhfR/u7gatAulKZo20I6Zs/TPAeu8wO8tSkKwFgvydCI8yw1WP1Sf/1zRYVH5R6
-s9SOuJwLDhnBLIC6i0etnh+OCf16I5wFHkgGvBAx+Ec23vCN5R5ZOaaHXMkogx19
-uPLm5nSt1Mj60HV7hTh30QTDR5urgcQ+OHbt+sYJnw+VyOJfP6XtlXOSAQhw/VL+
-klokrmYS4DhlIHrSnJTfpPymPhqsSVHd97Jr20nbuBbXlRsCAwEAAaNTMFEwHQYD
-VR0OBBYEFHMbgh0q1cxHVpR0UCWq92+0zQrsMB8GA1UdIwQYMBaAFHMbgh0q1cxH
-VpR0UCWq92+0zQrsMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEB
-AEJ554uXpkgXLGECVueikjzGOyCb7aBmKYdssuutWpEGM3pZpyCKzz/tsanULH3Q
-kNu5/eJYc0BBI2YXz1eIFrJR3fM+qbpIvP1j67CLhsjtvKSQTFEvqseCCZ1l34WK
-dNzpwf0HMB5MAt6T8xeQ5gDRNo7C3HuKT/8WouaIdn9H2UEluQBDM5Yg+cpDWOlv
-Dr/IzcLC/pyEmx7spkPfvxebtY64kdFh8qyl9iWZLO9wrpaWY1vV41sjZ1F41AgY
-KaRWGJhHwhyDcYzelsn0Ew2LdB7WfFQZfZe9EHJTj7bd6y6AbkduZ31oebXYlOzo
-1DkrU2K5aMlVF/vExWpfChk=
+MIIBkzCCAUUCFGipMcv8Oq6O89V+OkbybaF4q3XaMAUGAytlcDBsMQswCQYDVQQG
```

Comment:
what kind of certificate is this? and where did it come from (I know it's in a test)

Reply:
oh I see.